src/securejoin.rs

```diff
@@ -455,6 +455,8 @@ pub(crate) async fn handle_securejoin_handshake(
                     }
                     None => bail!("Chat {} not found", &field_grpid),
                 }
+                inviter_progress!(context, contact_id, 800);
```

Comment:
I think it can, and the tests will pass, but some products may expect this event. At least in deltachat.h it's documented that this event is generated for verified groups